src/css/styles.scss

```diff
@@ -1,46 +1,90 @@
-// Load CSS Reset from NPM
-// CSS resets "normalize" how browser styles are applied.
-// Try commenting this out and saving. What changes in the browser?
-@import "~normalize.css";
+@media screen and (min-width: 468px) and (max-width: 768px) {
```

Comment:
Do you actually want these styles only to start at a minimum of 468px? What about smaller iPhones, which are 320px - do these styles not apply here? Something to consider. I'd also check in with Yuluan, I think she worked on this area too. Be sure to do that before merging so you don't end up with any unpleasant surprises :)